app/models/push.rb

```diff
@@ -27,6 +28,24 @@ def ids_not_found
       push_ids - found_ids
     end
 
+    def validate_status
+      invalid_docs = export_search.solr_documents.reject do |doc|
+        doc.validation_status_for_aapb == [Asset::VALIDATION_STATUSES[:valid]]
+      end
+      return if invalid_docs.blank?
+
+      add_status_error(invalid_docs, Asset::VALIDATION_STATUSES[:missing_children])
+    end
+
+    def add_status_error(invalid_docs, status)
+      ids_matching_status = invalid_docs.select { |doc| doc.validation_status_for_aapb.include?(status) }.map(&:id)
+      # Prevents adding errors to docs that don't have a value
+      # in :validation_status_for_aapb, including all non-Assets.
+      return if ids_matching_status.blank?
```

Comment:
This means non-Asset records won't show an error (I _think_ only Assets get pushed anyways, so I'm not sure if this is even relevant).

More importantly, it means historic Assets that haven't been updated since this change won't throw errors. 

Is this what we want?